libs/notifications/package.json

```diff
@@ -0,0 +1,50 @@
+{
+  "name": "@novu/notifications",
```

Comment:
New package for all notification workflows to allow reusing them for triggering across the monorepo